src/app/ui-helper.js

```diff
@@ -21,6 +21,13 @@ module.exports = {
       cls);
```

Comment:
If we use input, it looks much more uniform. Also I don't think that anyone reads one of these outputs - they have to be reformatted before they are readable, so you have to copy-past them into a smart editor anyway.